examples/ASDKgram/Sample/Flickrgram.pch

```diff
@@ -0,0 +1,17 @@
+//
+//  ASDKgram.pch
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 2/26/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#ifndef Flickrgram_pch
+#define Flickrgram_pch
+
+#import <UIKit/UIKit.h>
```

Comment:
Here is where you could add Foundation and AsyncDisplayKit includes.  Also need to rename this file, although that may require adjusting the project build settinsg to be able to find the file.